src/org/Dictionary.java

```diff
@@ -18,22 +15,185 @@
+    k.put(Language.ENGLISH, "Go");
+    k.put(Language.PORTUGESE, "Go");
+    Entry go = new Entry(k);
+    this.addEntry("Go", go);
+
+    HashMap<Language, String> l = new HashMap<Language, String>();
+    l = new HashMap<Language, String>();
+    l.put(Language.SPANISH, "Finalizar");
+    l.put(Language.ENGLISH, "Done");
+    l.put(Language.PORTUGESE, "Concluído");
+    Entry done = new Entry(l);
+    this.addEntry("Done", done);
+
+    HashMap<Language, String> m = new HashMap<Language, String>();
+    m = new HashMap<Language, String>();
+    m.put(Language.SPANISH, "Mostrar en el mapa");
+    m.put(Language.ENGLISH, "Show on Map");
+    m.put(Language.PORTUGESE, "Mostrar no mapa");
+    Entry showOnMap = new Entry(m);
+    this.addEntry("Show on Map", showOnMap);
+
+    HashMap<Language, String> n = new HashMap<Language, String>();
+    n = new HashMap<Language, String>();
+    n.put(Language.SPANISH, "Ver información");
+    n.put(Language.ENGLISH, "View Info");
+    n.put(Language.PORTUGESE, "Ver informações");
+    Entry viewInfo = new Entry(n);
+    this.addEntry("View Info", viewInfo);
+
+    HashMap<Language, String> o = new HashMap<Language, String>();
+    o = new HashMap<Language, String>();
+    o.put(Language.SPANISH, "Cerrar sesión");
+    o.put(Language.ENGLISH, "Log off");
+    o.put(Language.PORTUGESE, "Terminar sessão");
+    Entry logOff = new Entry(o);
+    this.addEntry("Log off", logOff);
+
+    HashMap<Language, String> p = new HashMap<Language, String>();
+    p = new HashMap<Language, String>();
+    p.put(Language.SPANISH, "Editar Mapa");
+    p.put(Language.ENGLISH, "Edit Map");
+    p.put(Language.PORTUGESE, "Editar Mapa");
+    Entry editMap = new Entry(p);
+    this.addEntry("Edit Map", editMap);
+
+    HashMap<Language, String> q = new HashMap<Language, String>();
+    q = new HashMap<Language, String>();
+    q.put(Language.SPANISH, "Crear");
+    q.put(Language.ENGLISH, "Create");
+    q.put(Language.PORTUGESE, "Criar");
+    Entry create = new Entry(q);
+    this.addEntry("Create", create);
+
+    HashMap<Language, String> r = new HashMap<Language, String>();
+    r = new HashMap<Language, String>();
+    r.put(Language.SPANISH, "Eliminar");
+    r.put(Language.ENGLISH, "Delete");
+    r.put(Language.PORTUGESE, "Eliminar");
+    Entry delete = new Entry(r);
+    this.addEntry("Delete", delete);
+
+    HashMap<Language, String> t = new HashMap<Language, String>();
+    t = new HashMap<Language, String>();
+    t.put(Language.SPANISH, "Guardar");
+    t.put(Language.ENGLISH, "Save");
+    t.put(Language.PORTUGESE, "Guardar");
+    Entry save = new Entry(q);
+    this.addEntry("Save", save);
+
+    HashMap<Language, String> u = new HashMap<Language, String>();
+    u = new HashMap<Language, String>();
+    u.put(Language.SPANISH, "Cancelar");
+    u.put(Language.ENGLISH, "Cancel");
+    u.put(Language.PORTUGESE, "Cancelar");
+    Entry cancel = new Entry(u);
+    this.addEntry("Cancelar", cancel);
+
+
```

Comment:
The one letter variable names aren't very helpful for describing the data they are holding.  Especially since they are not the first letter of the English string they are representing